### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by aouaiti** [tommasoturchi/react-three-mind#10] [General]
This is not an issue report but a "thank you" message, this is precisely what I've been looking for !
however I would really appreciate the migration to mind-ar 1.2.0 since it allows the use of whatever three.js/r3f/r3d version
keep up the good work!

**Comment by tommasoturchi** [comment]
You're very welcome 😊
I'm working on the next release and migrating the library to vite, so it's better integrated with mind-ar and can then syncronise with its future releases. Stay tuned!
